specification/gedcom-03-datamodel.md

```diff
@@ -141,26 +186,19 @@ n HEAD                                     {1:1}
         +3 DATE <DateExact>                {0:1}  g7:DATE-exact
            +4 TIME <Time>                  {0:1}  g7:TIME
         +3 COPR <Text>                     {0:1}  g7:COPR
-  +1 DEST <Special>                        {0:1}  g7:DEST
-  +1 DATE <DateExact>                      {0:1}  g7:HEAD-DATE
-     +2 TIME <Time>                        {0:1}  g7:TIME
   +1 SUBM @<XREF:SUBM>@                    {0:1}  g7:SUBM
-  +1 COPR <Text>                           {0:1}  g7:COPR
-  +1 LANG <Language>                       {0:1}  g7:HEAD-LANG
-  +1 PLAC                                  {0:1}  g7:HEAD-PLAC
-     +2 FORM <List:Text>                   {1:1}  g7:HEAD-PLAC-FORM
-  +1 <<NOTE_STRUCTURE>>                    {0:1}
 ```
 
 The header pseudo-structure provides metadata about the entire dataset.
 A few substructures of note:
 
 - `GEDC` identifies the specification that this document conforms to.
+    It is recommended that `GEDC` be the first substructure of the header.
```

Comment:
If this is only a recommendation, it may be worth explaining this one, since the GEDC.VERS is used to determine how to parse the rest of the file, anything before it may not be parsed "correctly" per the version detection algorithm. Personally, I think it should be a _requirement_ in FamilySearch GEDCOM 7 that it be first.   It was already listed first so I might argue it was an ambiguity/clarification, not a breaking change.

Reply:
The recommendation was already in the spec; I just copied it up here from <https://gedcom.io/specifications/FamilySearchGEDCOMv7.html#GEDC>. I do think requiring it to be first would be a breaking change.